src/style/components/_hlist.styl

```diff
@@ -2,12 +2,16 @@
 
 .c-hlist
   clearfix()
+  &-items {
+    display: flex;
+    flex-direction: row;
+  }
   &-item
+    width: 100% !important;
     box-sizing: border-box
     float: left
```

Comment:
I guess you don't need **float and box-sizing** style any more?

Also I'm not sure how hlist should behave when it comes to "vertical align". For instance not all elements have same height and currently all elements inside list have _height: 100%_, which looks look ugly for some elements.

@Lanchi What do you thing of defining `align-items` option for hlist (since it's flex now). It should behave like align-item here: https://vuetifyjs.com/en/framework/grid#layout-playground

Reply:
You can keep it as is for now, needs to be defined better.